### DISCUSSION TITLE
Data usage

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#11793] [Help] [answered]
I want to give When you press the Search button Then the program will search for information related to what was typed in the Tag no. field.
![image](https://github.com/Budibase/budibase/assets/93240913/186be5b9-674b-4229-a5fa-59de24f25ece)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @LTCTHITA,/

This should be relatively easy to do, I'd recommend following the docs here; https://docs.budibase.com/docs/adding-a-search-field#1-using-the-table-block-the-easy-way.

Additionally heres another discussion were this has been answered. https://github.com/Budibase/budibase/discussions/10071#discussioncomment-5369143

I hope this helps.
